public/app/plugins/panel/dashlist/styles.ts

```diff
@@ -1,16 +1,16 @@
 import { css } from '@emotion/css';
 
-import { GrafanaTheme } from '@grafana/data';
-import { styleMixins, stylesFactory } from '@grafana/ui';
+import { GrafanaTheme2 } from '@grafana/data';
+import { styleMixins } from '@grafana/ui';
 
-export const getStyles = stylesFactory((theme: GrafanaTheme) => ({
+export const getStyles = (theme: GrafanaTheme2) => ({
   dashlistSectionHeader: css`
-    margin-bottom: ${theme.spacing.d};
-    color: ${theme.colors.textWeak};
+    margin-bottom: ${theme.spacing(1)};
```

Comment:
`dashlistSectionHeader` and `dashlistSection` both had the same`margin-bottom: theme.spacing.d` but now they have different values. Is it intentional? I think `theme.spacing.d == theme.spacing.md == theme.spacing(2)`

Reply:
Yeah you're right, it's supposed to be 2